modules/hcp-import-limitations.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-import.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-import-limitations_{context}"]
+= Limitations of managing imported hosted clusters
+
+Hosted clusters are automatically imported into the local {mce}, unlike a standalone {product-title} or third party clusters. Hosted clusters run some of their agents in the _hosted mode_ so that the agents do not use the resources of your cluster. If you choose to automatically import hosted clusters, you can update hosted clusters by using the `HostedCluster` resource on the management cluster.
+
+You can import hosted clusters into a location other than the local {mce-short} by using the {mce-short} or {rh-rhacm-first}. You must use the following options to update hosted clusters:
+
+* The console or the command-line interface of the management cluster, if you use the {mce} to import a hosted cluster.
+* A policy, if you use the {rh-rhacm} hub cluster to import a hosted cluster.
```

Comment:
How?

Reply:
We should remove this option of using ACM policy to update hosted clusters since we have not tested this and there is no example yet.